test/custom-cache.test.js

```diff
@@ -41,25 +44,24 @@ Object.keys(testAPIs).forEach(API => {
       })
     })
 
-    after(async () => {
-      if(orbitdb1)
+    after(() => {
+      // FIXME: shim to wait until the end of the event loop to call this
+      setTimeout(async () => {
         await orbitdb1.stop()
-
-      if (ipfsd)
         await stopIpfs(ipfsd)
+      }, 0)
     })
```

Comment:
what is this doing exactly. the function handed to after is completing before anything is actually done.
im sure theres a reason for this and im curious.

Reply:
adding a `setTimeout` with a time of 0 just makes sure the inner function is executed at the end of the event loop i.e. after all eventListeners are fired

Reply:
is this good practice for mocha tests or are there specific eventListeners you want to make sure are done?

Reply:
It's a hack, but likely a necessary one since I believe these eventListeners are at the IPFS level and not necessarily at the OrbitDB level

Reply:
Either way I'll remove the extraneous `setTimeout` calls at the `orbit-db` level since `orbit-db-test-utils` handles it